pkg/providers/aws/cluster_network_provider.go

```diff
@@ -63,6 +63,8 @@ const (
 	// see https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeVpcPeeringConnections.html
 	filterVpcPeeringRequesterId = "requester-vpc-info.vpc-id"
 	filterVpcPeeringAccepterId  = "accepter-vpc-info.vpc-id"
+	// default cidr values
+	defaultCIDRMask = "26"
```

Comment:
@laurafitzgerald not sure what I'm missing but I can't see this being used anywhere

Reply:
good spot, should be used lower down. will update.